Description: Don't monitor ipsec daemon
 For Ubuntu systemd will monitor the ovs-monitor-ipsec daemon so
 there is no need to spawn a separate monitor thread to deal with
 restarts.  Doing so has the side effect of confusing systemd into
 monitoring the wrong process.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

--- a/utilities/ovs-ctl.in
+++ b/utilities/ovs-ctl.in
@@ -245,7 +245,7 @@ start_ovs_ipsec () {
         --pidfile=${rundir}/ovs-monitor-ipsec.pid \
         --ike-daemon=$IKE_DAEMON \
         $no_restart \
-        --log-file --detach --monitor unix:${rundir}/db.sock || return 1
+        --log-file --detach unix:${rundir}/db.sock || return 1
     return 0
 }
